src/pages/Beta/Bridge/TabulationBox/QuestionBoxGeneral/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { StyledLogo, SeparatorBorder, StyledLogoMinus } from '../../styleds'
+import { Text } from '@pangolindex/components'
+import MinusLogo from 'src/assets/images/minus.png'
+import PlusLogo from 'src/assets/images/plus.png'
+import { SubCategories, useSubBridgeCategories } from 'src/state/bridge/hooks'
+
+type ISubCategory = {
+  subcategory: string
+}
+
+export const GeneralBox: React.FC<ISubCategory> = ({ subcategory }) => {
+  const [active, setActive] = useState<number>()
+  const [content, setContent] = useState<string>("")
+  const { data: categories } = useSubBridgeCategories(subcategory)
+
+  function activeLogo(key: number) {
+    if (active === key)
+      return (<StyledLogoMinus src={ MinusLogo } size={ '20px'} height={'4px'} />)
+    else
+      return <StyledLogo src={PlusLogo} size={'20px'} />
+  }
+
+  function activeText(key: number) {
+    if (active === key)
+    {
```

Comment:
formatting doesn't looks proper to me in some files, please run `yarn lint` so it will fix automatically.